script/nnbd_plugins.sh

```diff
@@ -7,6 +7,7 @@
 readonly NNBD_PLUGINS_LIST=(
   "android_intent"
   "battery"
+  "camera_platform_interface"
```

Comment:
This should be

```suggestion
  "camera"
```

I wasn't able to get it to run locally on stable, but this might help on the CI.

Reply:
So I was finally able to get the test running locally. The error with `google_maps_flutter` for the `build_all_plugins` test went away once this was changed.

Reply:
I believe the issue is that `camera` needs to be excluded from `stable` and `master` until both the `camera` and `camera_platform_interface` package are migrated. `camera` is in a weird spot where it conflicts with migrated and non-migrated plugins.

Let me know if this works

Reply:
Wow thanks @bparrishMines, this completely boggled me ;). 

I have added a new commit where "camera" is now listed in both the "NNBD_PLUGINS_LIST" and the "NON_NNBD_PLUGINS_LIST". I will keep an eye on the tree with fingers crossed ;).

Reply:
Nice stuff that did it, thanks so much @bparrishMines. I will merge this now and will create the NNBD PR for the camera package.